commit 8667b86d6da31dce148f78b95991b879f7b234f6
Author: Dave Stevenson <dave.stevenson@raspberrypi.org>
Date:   Thu May 2 14:30:24 2019 +0100

    staging: bcm2835-codec: Correct port width calc for truncation
    
    The calculation converting from V4L2 bytesperline to MMAL
    width had an operator ordering issue that lead to Bayer raw 10
    (and 12 and 14) setting an incorrect stride for the buffer.
    Correct this operation ordering issue.
    
    Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.org>

diff --git a/drivers/staging/vc04_services/bcm2835-codec/bcm2835-v4l2-codec.c b/drivers/staging/vc04_services/bcm2835-codec/bcm2835-v4l2-codec.c
index 4b67c9ea73e1..97b1f2b953ec 100644
--- a/drivers/staging/vc04_services/bcm2835-codec/bcm2835-v4l2-codec.c
+++ b/drivers/staging/vc04_services/bcm2835-codec/bcm2835-v4l2-codec.c
@@ -605,8 +605,8 @@ static void setup_mmal_port_format(struct bcm2835_codec_ctx *ctx,
 
 	if (!(q_data->fmt->flags & V4L2_FMT_FLAG_COMPRESSED)) {
 		/* Raw image format - set width/height */
-		port->es.video.width = q_data->bytesperline /
-						(q_data->fmt->depth >> 3);
+		port->es.video.width = (q_data->bytesperline << 3) /
+						q_data->fmt->depth;
 		port->es.video.height = q_data->height;
 		port->es.video.crop.width = q_data->crop_width;
 		port->es.video.crop.height = q_data->crop_height;
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1487_8667b86d6da31dce148f78b95991b879f7b234f6.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1487_8667b86d6da31dce148f78b95991b879f7b234f6.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
